packages/main/src/App.js

```diff
@@ -97,6 +98,7 @@ const App = () => (
           <Route exact path="/reset-password" component={ResetPassword} />
           <Route path="/change-password" component={ChangePassword} />
           <Route path="/whyzurichat" component={WhyZuriChat} />
+          <Route path="/errorpage" component={ErrorPage} />
```

Comment:
This is not how error pages are declared
it should be in this format 
 `<Route path="*" component={ErrorPage} />`
and should be the very last page declared under routes

Reply:
Alright, thanks. All changes will be made.